src/item_group.cpp

```diff
@@ -68,9 +68,10 @@ item Single_item_creator::create_single( const time_point &birthday, RecursionLi
     }
     if( modifier ) {
         modifier->modify( tmp );
+    } else {
+        tmp = tmp.in_its_container();
     }
     // TODO: change the spawn lists to contain proper references to containers
```

Comment:
you need to move the comment if you moved the line of code it's referring to.

Reply:
done